homework_3_objects/main.js

```diff
@@ -0,0 +1,79 @@
+// Duration between dates
+
+const durationBetweenDates = (startDate = '1 Jan 1970', endDate = Date(), units = 'days') => {
+  const start = new Date(startDate);
+  const end = new Date(endDate);
+
+  switch (units) {
+    case 'seconds':
+      return (Math.abs(start - end) / 1000).toFixed() + ' ' + units;
+    case 'minutes':
+      return (Math.abs(start - end) / 1000 / 60).toFixed() + ' ' + units;
+    case 'hours':
+      return (Math.abs(start - end) / 1000 / 60 / 60).toFixed() + ' ' + units;
+    case 'days':
+      return (Math.abs(start - end) / 1000 / 60 / 60 / 24).toFixed() + ' ' + units;
+    default:
+      return 'Please, specify correct units. Available options: "seconds", "minutes", "hours", "days"';
+  }
```

Comment:
Do you see a lot of copy/past here?
Let's try to avoid them.
1. Try to create a "map object" with keys - units and values - numbers.
2. This solution gives you the ability to avoid the `swithc-case` construction.

Reply:
I thought it was a bit weird, but I could not come up with a better solution. Thank you for the tip 🙌 I hope the changed code looks better now.